v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/event_bus.py

```diff
@@ -1,16 +1,30 @@
-from pyee import EventEmitter
+from pyee.asyncio import AsyncIOEventEmitter
 import log_wrapper
 from appdaemon.plugins.hass.hassapi import Hass
 
 
-class EventBus(EventEmitter):
+class EventBus(AsyncIOEventEmitter):
```

Comment:
As discussed before, I am not convinced that having one shared event bus for all events is the right choice.